core/src/test/java/hudson/model/FingerprintCleanupThreadTest.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi
```

Comment:
For a new file, you want to use the current year and the correct owner. (In your case `CloudBees, Inc.`) Similar example here: https://github.com/jenkinsci/jenkins/blob/be2dbec1cd14e03ddd54601ee3b2d581840bd1d4/core/src/main/java/jenkins/install/InstallState.java#L4